tools/extractors/extractors.cpp

```diff
@@ -35,12 +35,21 @@ void InitialGameStateExtractor::extract(GameState &state, Difficulty difficulty)
 	this->extractCityScenery(state, "ALIENMAP_", "alienmap", "alien", "stratmap", "loftemps",
 	                         "cityovr", state.cities[humanMapId]);
 
+	/*UString map = "08PORT";
```

Comment:
If this is debug/testing code that doesn't need to be in the git history maybe remove it?